packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -30,7 +30,7 @@
   },
   "scripts": {
     "compile": "webpack",
-    "lint": "tslint --project .",
+    "lint": "tslint --project . -e **/minSObjects.json",
```

Comment:
can we use a tslint config file?

Reply:
@xyc - That was my first choice and couldn't get it to work.